tests/interfaces/network.go

```diff
@@ -58,5 +58,6 @@ type Network interface {
 		sourceReceipt *types.Receipt,
 		source SubnetTestInfo,
 		destination SubnetTestInfo,
-		expectSuccess bool) *types.Receipt
+		expectSuccess bool,
```

Comment:
do you have an auto formatter for this btw?

Reply:
No unfortunately not. Just doing them as I see them